src/pkg/cli/deploymentsList_test.go

```diff
@@ -77,8 +78,8 @@ func TestDeploymentsList(t *testing.T) {
 		if err != nil {
 			t.Fatalf("DeploymentsList() error = %v", err)
 		}
-		expectedOutput := `Id      Provider    DeployedAt
-a1b2c3  playground  ` + timestamppb.Now().AsTime().Format("2006-01-02T15:04:05Z07:00") + `
+		expectedOutput := `Id      Provider    DeployedAt            Action
+a1b2c3  playground  ` + timestamppb.Now().AsTime().Format("2006-01-02T15:04:05Z07:00") + `  UP
```

Comment:
I don't think we should show "down" in this list, so I don't think we need this column at all